qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1044,7 +985,7 @@ def to_dataframe(self):
             meta = qdb.sql_connection.TRN.execute_fetchindex()
 
             # Create the dataframe and clean it up a bit
-            df = pd.DataFrame((list(x) for x in meta), columns=cols)
+            df = pd.DataFrame((list(x) for x in meta), columns=cols, dtype=str)
```

Comment:
The only case that I'm concerned here is if there is a `None` in `meta`:
```python
In [20]: l = [[1, 3], [None, 2]]

In [21]: df = pd.DataFrame(l, columns=['c1', 'c2'], dtype=str)

In [22]: df
Out[22]: 
    c1 c2
0    1  3
1  NaN  2
```
It changes the `None` to `NaN`. Can this case occur in here?